constraint-solver/src/solver/tests/test_utils.rs

```diff
@@ -0,0 +1,62 @@
+use std::collections::BTreeMap;
+
+use powdr_number::GoldilocksField;
+
+use crate::{quadratic_symbolic_expression::QuadraticSymbolicExpression, solver::Solver};
+
+pub type Var = &'static str;
+pub type Qse = QuadraticSymbolicExpression<GoldilocksField, Var>;
+
+pub fn var(name: Var) -> Qse {
+    Qse::from_unknown_variable(name)
+}
+
+pub fn constant(value: u64) -> Qse {
+    GoldilocksField::from(value).into()
+}
+
+pub fn assert_solve_result(
+    mut constraints: Vec<Qse>,
+    expected_assignments: Vec<(Var, GoldilocksField)>,
+) {
+    init_logging();
```

Comment:
Why not use the crate we use everwhere else?

Reply:
Ah nice! Using `test_log` now.